src/main.py

```diff
@@ -44,11 +52,17 @@ def main():
         logger.critical(f"No configuration file is found in {config_file_path}")
         exit(1)
 
-    parrotfish = Parrotfish(configuration)
+    if args.step_function:
+        step_function = StepFunction(configuration)
+        step_function.optimize()
+        pass
```

Comment:
I believe this is redundant. Remove if so

Reply:
This invokes StepFunction class, which handles the optimization of a step function.